troubleshooting/tools/requirement.txt

```diff
@@ -1 +1,2 @@
 prettytable>=0.7.2
+PTable>=0.9.2
```

Comment:
I don't see this imported, what does this new line accomplish?

Reply:
prettytable doesn't support **title** option for the table, for that I had to do a pip install PTable.
import is not required.

Reply:
Required for adding title to the tables, import is not needed. Did a pip install on my system.